backend/fleet_management/tests/test_auth_token.py

```diff
@@ -24,23 +24,23 @@ def test_successful_user_login(self):
             format="json",
         )
         self.assertEqual(driver_res.status_code, status.HTTP_200_OK)
-        self.assertContains(driver_res, "token")
+        self.assertContains(driver_res, "access")
```

Comment:
just a note here: we will need to support both options.
Older clients will send out requests with old "token", and so on the backend we have to accept that.